### PR TITLE
Update FITSImageData.ipynb

### PR DESCRIPTION
**Pull request by Prateekgidolia**
Set width in FITSProjection.

## PR Checklist

- [ ] Code passes flake8 checker
- [ ] New features are documented, with docstrings and narrative docs
- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

File: doc/source/visualizing/FITSImageData.ipynb [hunk 1]
```diff
@@ -79,7 +79,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "which took the same parameters as `ProjectionPlot` except the width, because `FITSProjection` and `FITSSlice` always make slices and projections of the width of the domain size, at the finest resolution available in the simulation, in a unit determined to be appropriate for the physical size of the dataset."
```
Comment (jzuhone) [inline doc/source/visualizing/FITSImageData.ipynb]: I think the text you deleted is fine here, but I also think that you could maybe add a quick example to this file of making the same projection with a different width, which would be clearer.

Comment (Prateekgidolia) [reply doc/source/visualizing/FITSImageData.ipynb]: Yes, I added a quick example, set a new pull request.

**Comment by Prateekgidolia** [comment]
yes sure.

**Comment by jzuhone** [comment]
Closed this in favor of PR #1913

**Pull request closed without merging**